### DISCUSSION TITLE
Watermark image with image?

### DISCUSSION THREAD

**Discussion by OpenSpacesAndPlaces** [SixLabors/ImageSharp#1733] [Q&A] [answered]
Is it possible to watermark an image with another image? The samples and mds only show using fonts.

Thanks!

**Comment by JimBobSquarePants** [comment, marked as answer]
https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Processing.DrawImageExtensions.html

**Reply by OpenSpacesAndPlaces** [reply]
Thanks - I'll take a look!
